vault/testing.go

```diff
@@ -914,6 +921,46 @@ func (c *TestCluster) Start() {
 	if c.SetupFunc != nil {
 		c.SetupFunc()
 	}
+
+WAITACTIVE:
+	for i := 0; i < 60; i++ {
+		for _, core := range c.Cores {
+			if standby, _ := core.Core.Standby(); !standby {
+				break WAITACTIVE
```

Comment:
I recognise that this is not likely or a great state to be in, but if we're not considered active by 60 seconds for whatever reason, should we error out?

I think if that happens, the error we'd likely get is `error setting up global rate limit quota`, which might be confusing (as that wasn't the real problem).

Reply:
Totally, I think that was my intent when I wrote this.